provisioning/plugins/corim-psa-decoder/ccaplatformconfigid.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package main
+
+import (
+	"fmt"
+
+	"github.com/veraison/corim/comid"
+	structpb "google.golang.org/protobuf/types/known/structpb"
+)
+
+type CCAPlatformConfigID struct {
+	Label string
+	Value []byte
+}
+
+func (o *CCAPlatformConfigID) FromMeasurement(m comid.Measurement) error {
+	id, err := m.Key.GetCCAPlatformConfigID()
+	if err != nil {
+		return fmt.Errorf("failed extracting mkey for cca-platform-config-id: %w", err)
+	}
+	o.Label = string(id)
+
+	if m.Val.RawValue == nil {
+		return fmt.Errorf("no CCA platform config id set in the measurements")
+	}
+	r := *m.Val.RawValue
+
+	o.Value, err = r.GetBytes()
+	if err != nil {
+		return fmt.Errorf("failed to get CCA platform config id: %w", err)
+	}
+	return nil
+}
+
+func (o CCAPlatformConfigID) MakeRefAttrs(c PSAClassAttributes) (*structpb.Struct, error) {
+	refAttrs := map[string]interface{}{
+		"psa.impl-id":               c.ImplID,
```

Comment:
as discussed f2f, we might want to have impl-id and other in their own `cca.` namespace

Reply:
and add a "_type" key too to the attributes map that allows the consumer to distinguish swcomp from config stuff

Reply:
> Yes, I am working on Phase -2 of this change!. It is very difficult to manage large set of changes in one lot, so I will  have it into two parts!

Reply:
ok -- not sure I understand what phase-2 is though :)

Reply:
this is phase -1 of Provisioning. We will have a next phase of Provisioning changes as Phase-2

Reply:
I raised Issue #53 to complete CCA provisioning side of thing